src/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Bernard\Exception;
+
+/**
+ * Interface for exceptions thrown by Bernard
+ * @package Bernard
+ */
+interface ExceptionInterface
```

Comment:
Ha! I spend my days in PSR-land, but I (mostly) agree with any reasonably argued opinion. Will fix.